dev_guide/downward_api.adoc

```diff
@@ -222,3 +219,139 @@ annotation2=456
+=== Using Environment Variables
+
+. When creating a pod configuration, specify environment variables that
+correspond to the contents of the `resources` field in the `spec.container`
+field:
++
+====
+----
+...
+spec:
+  containers:
+    - name: test-container
+      image: gcr.io/google_containers/busybox:1.24
+      command: [ "/bin/sh", "-c", "env" ]
+      resources:
+        requests:
+          memory: "32Mi"
+          cpu: "125m"
+        limits:
+          memory: "64Mi"
+          cpu: "250m"
+      env:
+        - name: MY_CPU_REQUEST
+          valueFrom:
+            resourceFieldRef:
+              resource: requests.cpu
+        - name: MY_CPU_LIMIT
+          valueFrom:
+            resourceFieldRef:
+              resource: limits.cpu
+        - name: MY_MEM_REQUEST
+          valueFrom:
+            resourceFieldRef:
+              resource: requests.memory
+        - name: MY_MEM_LIMIT
+          valueFrom:
+            resourceFieldRef:
+              resource: limits.memory
+...
+----
+====
++
+If the resource limits are not included in the container configuration, the downward API defaults to the node's CPU and memory capacities.
+
+. Create the pod from the `*_pod.json_*` file:
++
+====
+----
+$ oc create -f pod.json
+----
+====
+
+. To verify that the pod has been created with the expected values, you can grep the pod's logs and look for the following variables:
++
+- `*MY_MEM_LIMIT*`
+- `*MY_CPU_LIMIT*`
+- `*MY_MEM_REQUEST*`
+- `*MY_CPU_REQUEST*`
+
+[[dapi-resources-volume-plugin]]
+=== Using the Volume Plug-in
+
+. When creating a pod configuration, use the `spec.volumes.downwardAPI.items` field to describe the desired resources that correspond to the `spec.resources` field:
++
+====
+----
+....
+spec:
+  containers:
+    - name: client-container
+      image: gcr.io/google_containers/busybox:1.24
+      command: [ "/bin/sh", "-c", "env" ]
+      resources:
+        requests:
+          memory: "32Mi"
+          cpu: "125m"
+        limits:
+          memory: "64Mi"
+          cpu: "250m"
+      volumeMounts:
+        - name: podinfo
+          mountPath: /etc
+          readOnly: false
+  volumes:
+    - name: podinfo
+      downwardAPI:
+        items:
+          - path: "cpu_limit"
+            resourceFieldRef:
+              containerName: client-container
+              resource: limits.cpu
+          - path: "cpu_request"
+            resourceFieldRef:
+              containerName: client-container
+              resource: requests.cpu
+          - path: "mem_limit"
+            resourceFieldRef:
+              containerName: client-container
+              resource: limits.memory
+          - path: "mem_request"
+            resourceFieldRef:
+              containerName: client-container
+              resource: requests.memory
+....
+----
+====
++
+If the resource limits are not included in the container configuration, the downward API defaults to the node's CPU and memory capacities.
+
```

Comment:
node's CPU and memory allocatables

It was changed in https://github.com/kubernetes/kubernetes/pull/29639 and I also sent a doc update PR here:
https://github.com/kubernetes/kubernetes.github.io/pull/1107/files